.github/workflows/push.yml

```
@@ -0,0 +1,25 @@
+name: プッシュ時ビルド
+
+on:
+  push:
+    branches:
+      - 'feature/**'
+
+jobs:
+    pull_request:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v4
+              with:
+                fetch-depth: 0
+            - name: PostgreSQL環境の構築
+              run: docker compose up -d
+            - name: Java環境の構築
+              uses: actions/setup-java@v4
+              with:
+                distribution: corretto
+                java-version: 17
+            - name: Build
```

明示的に Build と Test を step として分けてもらっていいですか?

```
```suggestion
            - name: Build
            run: ./gradlew build -xtest
```

みたいな感じ

---

元のコミットに含めて修正しました。
./gradlew testをその後段に入れてます